### ISSUE TITLE
Changing app icon

### ISSUE THREAD

**Issue by abdelillahbel** [devunionorg/ColorMakerMAUI#2]
I am going to change app icon

**Comment by amuza2** [comment]
ok amazing💪, I think it's better if we have a variation of icons to choose from.
what do you think?

**Comment by abdelillahbel** [comment]
yeah, I will try to implement a method that let user choose from a variation of icons.
